### ISSUE TITLE
No mappings in the git blame window

### ISSUE THREAD

**Issue by zenofsahil** [tpope/vim-fugitive#2015]
I'm using MacVim with vim-fugitive installed with Vundle. Facing the same issue as mentioned in #1835. When I run `:Git blame` no mappings work in the blame window. Pressing `o` just enters into a new line. 

I checked the output of `:scriptnames` and `ftplugin/fugitiveblame.vim` is present there. 

Could use some help with this.

**Comment by tpope** [comment]
Does your vimrc have `filetype plugin indent on`?

**Comment by zenofsahil** [comment]
Thanks @tpope. Yes, `filetype plugin indent on` is present in my vimrc.

**Comment by tpope** [comment]
The real problem is `'modifiable'` is on, otherwise you'd get `E21: Cannot make changes, 'modifiable' is off` when pressing `o`. Check `:verbose set modifiable?` and see if it tells you that some other plugin turned it off.

**Comment by zenofsahil** [comment]
`:verbose set modifiable?` returns ` modifiable`. I checked and this only seems to be happening with MacVim. If I start vim in a terminal session, the mappings work as expected.

**Comment by tpope** [comment]
Check `:echo FugitiveResult(bufnr(''))` in the blame buffer. We're expecting a dictionary with various context about the blame that was run. If it's an empty dictionary, that's a problem.

Compare to your working terminal setup, too.

**Comment by zenofsahil** [comment]
`:echo FugitiveResult(bufnr(''))` resulted in an empty dictionary as compared to running the same command inside the terminal vim instance which had the context. I opened a some other repos and ran `:Git blame`. The issue seems to be there only with Rust files. 

I commented out `Plugin 'rust-lang/rust.vim'` within the vundle initialization settings in my vimrc and the mappings started working again. Looks like this is the plugin causing the issue. I tried looking into the plugin but wasn't able to pin down what might be causing this. Any pointers?

**Comment by tpope** [comment]
I would start by commenting out the files in `plugin/`, `ftplugin/`, and `indent/` and see if that fixes it. If so, start uncommenting stuff and see if you can narrow down what's needed to break it.

**Comment by zenofsahil** [comment]
I removed the `ftplugin` directory inside the `rust.vim` plugin. Mappings worked as expected then. Weird thing is that after I put the `ftplugin` directory back, the mappings continued to work correctly. So everything is working correctly now. All I did was remove the `ftplugin` directory, run MacVim, and put the `ftplugin` directory back where it was.

Appreciate your help @tpope. I'll test things out a bit more since I'm still not sure what the exact issue was. If things continue to work correctly over the next couple of days, I'll close the issue. Thanks!
